addon/components/phone-input.js

```diff
@@ -253,9 +253,9 @@ export default Component.extend({
 
     if (selectedCountry.iso2) {
       this._iti.setCountry(selectedCountry.iso2);
-    }
 
-    this.input();
+      this.input();
```

Comment:
I agree this change makes total sense. I should have catched during the previous review. Sorry.

That said - I checked out the changes and I cannot reproduce the bug you're reporting. Do you think you could push a commit with a failing test and then fix it?